spec/Menu_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'Menu'
+
+describe Menu do
+
+  subject(:menu) { Class.new { extend Menu } }
```

Comment:
I have never seen this extend thing before, guessing it just creates a new Class yeah?
Another way you can write this is:
subject(:menu) { described_class.new }